tests/test_locking.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+from unittest import mock
+import os
+import meg_runtime.locking as locking
+from meg_runtime.locking.lockFile import LockFile
+
+TOTAL_LOCKS_INITALIZED = 3
+
+@pytest.fixture()
+def generateLocking():
+    try:
```

Comment:
Same thing as the other tests don't catch exceptions, you can look at how I made directories, I think you just need the ignore_exists. If you can't make the directory it should throw an exception.